scripts/publish_video/helper.js

```diff
@@ -360,6 +422,26 @@ async function shouldSkip(video) {
   }
 }
 
+async function isVideoAlreadyPostedOnHive(video) {
+  if (video.status === 'publish_manual') {
+    if (video.fromMobile === true ) {
+      try {
+        let doWeHavePostingAuthority = await hasPostingAuthority(video.owner);
```

Comment:
How does checking posting authority have anything to do to being on chain? Plus it doesn't seem there are any checks for on chain here either

Reply:
my bad.

Reply:
@sag333ar What do you intend to do here going forward?

Reply:
It's a function which was the result of copy-paste but forgot to remove it. I just updated code & removed this method.